### PR TITLE
Script netstat Monitoring

### PR DESCRIPTION
**Pull request by eltalkarim**
This is a script to deploy monitoring tools for our SUT.

## Description

We need this to query metrics from our SUT

## Related Issue

#78 

## Motivation and Context
Our algorithm needs to read the network stat to take decision 

## How Has This Been Tested?
By testing it on the SUT while hitting it with caliper

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist contributor:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: bin/deploy-sut.sh [hunk 1]
```diff
@@ -106,9 +106,9 @@ for index in ${!INSTANCE_LIST[@]}; do
     gcloud compute ssh ${USERNAME}@${INSTANCE_LIST[index]} --command "geth --nousb --datadir .ethereum/ init genesis.json"
     echo
     echo GENESIS INITIALISED on ${INSTANCE_LIST[index]}
-    ACCOUNT=$(gcloud compute ssh ${USERNAME}@${INSTANCE_LIST[index]} --command "geth --nousb --datadir .ethereum/ account list | cut -d "{" -f2 | cut -d "}" -f1")
+    ACCOUNT=$(gcloud compute ssh ${UAME}@${INSTANCE_LIST[index]} --command "geth --nousb --datadir .ethereum/ account list | cut -d "{" -f2 | cut -d "}" -f1")
```
Comment (fayanpardo) [inline bin/deploy-sut.sh]: Why the change from USERNAME to UAME?

**Review by fayanpardo — COMMENTED** [review]
Nice! Please check my comment in deploy-sut.sh

File: bin/deploy-sut.sh (hunk 1 above)
Comment (eltalkarim) [reply bin/deploy-sut.sh]: sorry got it fixed

**Review by fayanpardo — APPROVED** [review]
GJ!

**Review by SantoshDhirwani — APPROVED** [review]
